<ui version="4.0" >
 <author>Peter Hedlund</author>
 <class>FromToEntryPageForm</class>
 <widget class="QWidget" name="FromToEntryPageForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>326</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>-</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="direc_label" >
     <property name="title" >
      <string>Properties From Original</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="2" >
         <widget class="QLineEdit" name="fauxami_line" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="grade_label" >
          <property name="text" >
           <string>&amp;Confidence Level:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>gradebox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="KComboBox" name="gradebox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="fauxami_label" >
          <property name="text" >
           <string>&amp;False friend:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>fauxami_line</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <widget class="QGroupBox" name="GroupBox3" >
        <property name="title" >
         <string>Practice &amp;Counts</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="badCountEdit" >
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>999999999</number>
           </property>
           <property name="value" >
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="totalCountEdit" >
           <property name="maximum" >
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="bcount_label" >
           <property name="text" >
            <string>&amp;Wrong:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>badCountEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="qcount_label" >
           <property name="text" >
            <string>Tot&amp;al:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy" >
            <cstring>totalCountEdit</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="GroupBox2" >
        <property name="title" >
         <string>&amp;Last Practiced</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QPushButton" name="never" >
           <property name="text" >
            <string>&amp;Never</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QPushButton" name="today" >
           <property name="text" >
            <string>T&amp;oday</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2" >
          <widget class="QDateEdit" name="queryDateEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops" >
            <bool>false</bool>
           </property>
           <property name="toolTip" >
            <string>The date this expression was last practiced</string>
           </property>
           <property name="specialValueText" >
            <string>Not Practiced Yet</string>
           </property>
           <property name="dateTime" >
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>1999</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="minimumDate" >
            <date>
             <year>1999</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="displayFormat" >
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetGradesButton" >
       <property name="text" >
        <string>&amp;Reset Confidence Levels</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>421</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gradebox</tabstop>
  <tabstop>fauxami_line</tabstop>
  <tabstop>queryDateEdit</tabstop>
  <tabstop>today</tabstop>
  <tabstop>never</tabstop>
  <tabstop>totalCountEdit</tabstop>
  <tabstop>badCountEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
